### PR TITLE
Branch 23.06 resolve merge conflict for forward merge

### PR DESCRIPTION
**Pull request by alexbarghi-nv**
Resolves merge conflict

### CODE REVIEW

**Review by BradReesWork — APPROVED** [review]

**Comment by galipremsagar** [comment]
Not sure why but seems like there's additional commits in https://github.com/rapidsai/cugraph/pull/3408, that aren't showing up here.

**Comment by galipremsagar** [comment]
@alexbarghi-nv I think we will need to perform an no-squash merge from `branch-23.04` to `branch-23.06`, I tried doing it but there are quite a bit of conflicts that I'm not familiar of. I'm happy to hop on a call with you if you need help.

**Comment by alexbarghi-nv** [comment]
I know how to do the merge, just got out a meeting now.  I'll take care of it.

**Comment by galipremsagar** [comment]
> I know how to do the merge, just got out a meeting now. I'll take care of it.

Thanks @alexbarghi-nv !

**Comment by alexbarghi-nv** [comment]
@galipremsagar all the changes should be pulled in now and conflicts resolve.

File: .github/workflows/pr.yaml [hunk 1]
```diff
@@ -121,9 +121,9 @@ jobs:
       build_type: pull-request
       package-name: cugraph
       # Always want to test against latest dask/distributed.
-      test-before-amd64: "cd ./datasets && bash ./get_test_data.sh && cd - && RAPIDS_PY_WHEEL_NAME=pylibcugraph_cu11 rapids-download-wheels-from-s3 ./local-pylibcugraph-dep && pip install --no-deps ./local-pylibcugraph-dep/*.whl && pip install git+https://github.com/dask/dask.git@2023.3.2 git+https://github.com/dask/distributed.git@2023.3.2.1 git+https://github.com/rapidsai/dask-cuda.git@branch-23.04"
+      test-before-amd64: "cd ./datasets && bash ./get_test_data.sh && cd - && RAPIDS_PY_WHEEL_NAME=pylibcugraph_cu11 rapids-download-wheels-from-s3 ./local-pylibcugraph-dep && pip install --no-deps ./local-pylibcugraph-dep/*.whl && pip install git+https://github.com/dask/dask.git@main git+https://github.com/dask/distributed.git@main git+https://github.com/rapidsai/dask-cuda.git@branch-23.06"
```
Comment (galipremsagar) [inline .github/workflows/pr.yaml]: Lets unpin dask all repos at a time. Unpinning just cugraph might have unintended consequences.

Comment (alexbarghi-nv) [reply .github/workflows/pr.yaml]: @galipremsagar should be fixed now

**Review by galipremsagar — APPROVED** [review]
Thanks @alexbarghi-nv !

**Review by raydouglass — APPROVED** [review]

**Review by BradReesWork — APPROVED** [review]

**Review by ChuckHastings — APPROVED** [review]

**Pull request merged**